scripts/pgdata.sh

```diff
@@ -0,0 +1,50 @@
+#!/hab/pkgs/core/bash/5.1/20210826055113/bin/bash
```

Comment:
is this linking correct, this version might not be installed in new machines later, or even now.

Reply:
changed to /bin/bash